index.html

```diff
@@ -65,10 +65,33 @@ <h2>preferCurrentTab</h2>
         </dt>
         <dd>
           <p>
-            When {{MediaDevices/getDisplayMedia()}} is called, if <code>preferCurrentTab</code> is
-            <code>true</code>, the user agent SHOULD present the current tab as the most prominent
-            option. Otherwise, no such requirement is made.
+            When {{MediaDevices/getDisplayMedia()}} is called with <code>|constraints|</code> (a
+            dictionary), the user agent MUST run the following steps:
           </p>
+          <ol>
+            <li>
+              If <code>|constraints|.preferCurrentTab</code> is <code>false</code>, abort these
+              steps and continue instead with the
+              <a href="https://www.w3.org/TR/screen-capture/#dom-mediadevices-getdisplaymedia"
+                >getDisplayMedia algorithm</a
+              >.
+            </li>
+            <li>
+              If <code>|constraints|.</code>{{DisplayMediaStreamOptions/selfBrowserSurface}} is
+              {{SelfCapturePreferenceEnum/"exclude"}}, return a promise [=reject|rejected=] with a
+              newly [=exception/created=] {{TypeError}}.
+            </li>
+            <li>
+              All other steps associated with
+              <a href="https://www.w3.org/TR/screen-capture/#dom-mediadevices-getdisplaymedia"
```

Comment:
You probably want to turn the steps you want to run in that section into an algorithm and properly link to them here (with a return value that would enable you to avoid coditions in prose). I'm fine with a TODO to that effect, as it requires a refactoring on that spec's end.

Reply:
TODO added; does it work out?

Reply:
Maybe instead, you can do here something like "Call `getdisplaymedia`, while altering step 9.3 to prominently present the current tab..." ?

Reply:
I could, but I'm afraid it might not age well, as those steps can change. So that's one option.
Another option is:
> The rest of the getDisplayMedia algorithm should now be executed normally, with one change. If the algorithm's execution reaches the step where the user agent asks the user to choose a display surface to capture, the user agent SHOULD present the current tab as the most prominent option."

Wdyt?

Reply:
Is this LGTM?

Reply:
(I've integrated the change into the PR now.)